src/test.js

```diff
@@ -647,8 +658,11 @@ export function only( testName, callback ) {
 		return;
 	}
 
-	config.queue.length = 0;
-	focused = true;
+	// Only empty the queue in case this test is not part of a skipped module
+	if ( !config.currentModule.skip ) {
```

Comment:
Yeah, we should likely add a test like that as a separate module, but that can happen in another PR.